Author: Thomas Faughnan <tom@tjf.sh>
URL: https://github.com/tmux-plugins/tmux-resurrect/pull/455 (awaiting upstream merge)
Summary: Fix ps arguments to work for busybox
diff --git a/save_command_strategies/ps.sh b/save_command_strategies/ps.sh
index 544426c..15bb5aa 100755
--- a/save_command_strategies/ps.sh
+++ b/save_command_strategies/ps.sh
@@ -11,7 +11,7 @@ exit_safely_if_empty_ppid() {
 }
 
 full_command() {
-	ps -ao "ppid command" |
+	ps -ao "ppid,args" |
 		sed "s/^ *//" |
 		grep "^${PANE_PID}" |
 		cut -d' ' -f2-
